src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -441,6 +445,49 @@ public static String calcMD5(final byte[] bytes) {
     }
 
     /**
+     * Calculates the MD5 for the specified file and returns it as a String
+     *
+     * @param file file whose MD5 to calculate
+     * @return file's MD5 in String form
+     * @throws RuntimeException if the file could not be read
+     */
+    public static String calculateFileMD5( final File file ) throws IOException{
+        return Utils.calcMD5(getBytesFromFile(file));
+    }
+
+    /**
+     * Returns the byte[] of the entire contents of file, for md5 calculations
+     */
+    private static byte[] getBytesFromFile(final File file) throws IOException {
```

Comment:
Actually, I take it back. Please delete this method completely, and replace all usages with calls to `org.apache.commons.io.FileUtils.readFileToByteArray()`